src/utils/mailConfigurations.js

```diff
@@ -0,0 +1,14 @@
+export class MailConfigurations{
+    static emailVerificationConfig(email, token){
+        const config= {
+        from: 'vikingsatlp@gmail.com',
+        to: `${email}`,
+        subject: 'Vikings email verification',
+        text: `Hi! Welcome to Vikings e-commerce.
+        Please follow the given link to activate your account
+        ${process.env.VERIFY_URL}/${token} 
+        Thank you!`
+       };
+       return config;
+    };
+};
```

Comment:
Let's have a good email template. check this link https://codepen.io/rKalways/pen/RwwQGKd